src/server/pachyderm_test.go

```diff
@@ -170,8 +170,9 @@ func TestPipelineWithLargeFiles(t *testing.T) {
 
 	commit1, err := c.StartCommit(dataRepo, "master")
 	require.NoError(t, err)
+	chunkSize := int(pfs.ChunkSize / 32) // We used to use a full ChunkSize, but it was increased which caused this test to take too long.
```

Comment:
Changing the `pfs.ChunkSize` made this test take way too long, so I lessened it to compensate.